VAGRANT.md

```diff
@@ -97,13 +111,7 @@ To run specific tests you can add tags just before the `Scenario line`, e.g.
 
 and then
 
-    NOMINATIM_SERVER=http://localhost:8089/nominatim lettuce -t bug-34
-
-
-## Running unit tests
-
-    cd ~/Nominatim/tests/php
-    phpunit ./
+    behave -DBUILDDIR=/home/vagrant/build/ --tags @bug-34
```

Comment:
With behave you can now just state the line in the file to run a single test, e.g like that:

    behave -DBUILDDIR=/home/vagrant/build/ features/api/reverse.feature:34

Very useful feature.

Reply:
Nice, just like Ruby's rspec